### ISSUE TITLE
RSA error when using key files with CRLF

### ISSUE THREAD

**Issue by slorello89** [Vonage/vonage-dotnet-sdk#234]
### General description of the issue or question
If you try to create your Credentials object from the [private key path](https://github.com/Vonage/vonage-dotnet-sdk/blob/master/Vonage/Request/Credentials.cs#L89), if the private key uses CRLF line endings it will break.

This might arise if you used the Nexmo CLI to create your application and then copy-pasted the printed key into a notepad instance and did a naive save. You can work around this by running `dos2unix` against the key file to switch it to line feed. Regardless, it should be smart enough to replace CRLF with LF and continue processing.

### Which target framework(s) of .NET can you reproduce this issue against?
Any

### What operating system(s) is exhibiting this issue?
Windows

### Expected Behavior
Private Key's using CRLF line endings should work

### Actual Behavior
If a Private Key file is used with CRLF endings the key generator will throw a `System.ArgumentException: RsaUsingSha alg expects key to be of RSA type.` exception

### Steps to Reproduce
Follow "Create our Vonage Application" header's instructions in https://www.nexmo.com/blog/2020/08/07/how-to-play-audio-into-a-call-with-asp-net-core-mvc-dr - make sure the private.key is saved in a CRLF file, try running through the rest of the blog post, it will fail when you try to make a call using the Vonage Voice API.

**Comment by gagandeepp** [comment]
assign this to me

**Comment by gagandeepp** [comment]
How to test this?

**Comment by slorello89** [comment]
Here's a [Key](https://gist.github.com/slorello89/a162c11b10b72d2b163f8c0e556148b9) file that threw the exception for me.

**Comment by gagandeepp** [comment]
I have debugged this issue and found this link 
https://stackoverflow.com/questions/26551116/rsaprovider-importparameters-bad-data-for-some-special-rsa-private-keys

Code is throwing exception at   RSA.ImportParameters(RSAparams); in PemParse.cs file

**Comment by slorello89** [comment]
Hi @gagandeepp - I looked a bit deeper - the CRLF was actually not the problem. The issue was with the key itself.

The modulus component of the key is 256 bytes, however, the private exponent of the key that's encoded is only 255 bytes. The runtime made a change https://github.com/dotnet/corefx/pull/39087/files#diff-a6640d288f1f2114f5998225c0de2ff658318f8217377b05951ef4ce59dee5cbR79 that made this illegal. 

I did a bit more digging and it turns out that the private exponent advertised by the private key is in fact not the modular multiplicative 
inverse of (p-1)(q-1) thus I'm not sure if it's truly 'valid'. Going to close this issue as it's in fact not accurate - however, I'm going to be closing your PR as `hacktoberfest-accepted` so that you get credit for the contest. Thank you for contributing!
